lib/Targets/BufferDescriptorAnalysis.cpp

```diff
@@ -0,0 +1,391 @@
+//===- BufferDescriptorAnalysis.cpp -----------------------------*- C++ -*-===//
+//
+// This file is licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// (c) Copyright 2023 Xilinx Inc.
```

Comment:
Advanced Micro Devices, Inc.

Reply:
Will update the name here. Also, git checks show errors regarding clang formatting, and compilation warning. Will fix them as well.